pkg/volume/cinder/attacher_test.go

```diff
@@ -470,7 +470,7 @@ func (testcase *testcase) AttachDisk(instanceID, volumeID string) (string, error
 		return "", errors.New("unexpected AttachDisk call: wrong instanceID")
 	}
 
-	klog.V(4).Infof("AttachDisk call: %s, %s, returning %q, %v", volumeID, instanceID, expected.retDeviceName, expected.ret)
+	klog.V(4).InfoS("AttachDisk call volumeID and instanceID, returns retDeviceName and ret", "volumeID", volumeID, "instanceID", instanceID, "retDeviceName", expected.retDeviceName, "ret", expected.ret)
```

Comment:
Using klog in unittest doesn't seem to be correct. During tests logs should be captured and  so they can be written only when tests fails. I don't think this is supported with such calls to klog, that's why author used `klog.V(4)` to hide logs by default. Overall it's not correct to migrate those logs to InfoS as they should not be using klog at all.

Please change original calls from `klog.V(4).Infof` to `testcase.t.Logf`

Reply:
Hi, @serathius. The  `klog.V(4).Infof` in file `attacher_test.go` have been replaced with `testcase.t.Logf`. Waiting for your review with pleasure.